config/locales/en.yml

```diff
@@ -78,6 +78,8 @@ en:
       you_are_member: 'You are a member'
     show:
       users_of_events: 'List of users of this event:'
+      event_status: 'Status'
```

Comment:
You add this only for show action not for index. But we don't want duplication here, so I suggest to move it to event attributes (this from model). This make sense because status will behave like normal event attribute for our views.